kamailio/default.cfg

```diff
@@ -323,15 +326,31 @@ route[HANDLE_NOTIFY]
 route[HANDLE_MESSAGE]
 {
     if (is_method("MESSAGE")) {
-        if (isflagset(FLAG_INTERNALLY_SOURCED)) {
+        xlog("L_INFO", "$ci|MESSAGE from $fu to $tu");
+        if (isflagset(FLAG_INTERNALLY_SOURCED) || src_ip == myself) {
+           xlog("L_INFO", "$ci| routing MESSAGE to external from $fu to $tu");
+             
             route(INTERNAL_TO_EXTERNAL_RELAY);
+        #!ifdef TRAFFIC-FILTER-ROLE
+        } else if (!isflagset(FLAG_TRUSTED_SOURCE)
+            && $rd =~ "[0-9]{1,3}\.[0-9]{1,3}.[0-9]{1,3}\.[0-9]{1,3}") {
+            xlog("L_WARN", "$ci|end|dropping MESSAGE request with IP domain");
+        #!endif
         } else {
-            sl_send_reply("202", "delivered to /dev/null");
```

Comment:
I would like to keep existing behavior it the role is not enabled.  This was added after someone sent a message the was retried between FreeSWITCH servers for more than a week....